docs/docs_source/troubleshooting.rst

```diff
@@ -0,0 +1,100 @@
+reproducing an issue, it will record all HTTP requests and responses. You can then save the Fiddler trace and share
+it with Microsoft engineers. Fiddler traces are an invaluable tool when troubleshooting problems because they
+provide a full record of the HTTP traffic between the browser and Office Online. As a rule of thumb, hosts should
+always provide a Fiddler trace when reporting Office Online integration issues to Microsoft.
+
+..  _Fiddler: http://www.telerik.com/fiddler
+
+
+..  _Fiddler HTTPS:
+
+Enabling HTTPS decryption in Fiddler
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+Because Office Online traffic is encrypted, Fiddler must be configured to decrypt the HTTPS traffic in order to be
+useful. In order to enable HTTPS encryption in Fiddler, do the following:
+
+#. From Fiddler, click :menuselection:`Tools --> Fiddler Options...` to open the options dialog.
+#. On the :guilabel:`HTTPS` tab, check the :guilabel:`Capture HTTPS CONNECTs` check box.
+#. Check the :guilabel:`Decrypt HTTPS traffic` check box. When you do this Fiddler will display a dialog asking if you
+   wish to trust the Fiddler Root certificate. Click :guilabel:`Yes`. You may also see some security warnings from the
+   operating system asking if you want to install the certificate. Click :guilabel:`Yes` to all of these prompts.
+#. In the drop-down, select :guilabel:`...from browsers only`.
+#. Click :guilabel:`OK` in the options dialog.
+#. Close Fiddler and restart it.
+
+Fiddler is now configured to decrypt HTTPS traffic.
+
+..  figure:: /images/fiddler_https.png
+    :alt: An image showing the HTTPS decryption configuration UI in Fiddler.
+
+    Fiddler must be configured to decrypt HTTPS traffic in order to produce useful traces
+
+
+Using Fiddler to trace a session
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+Using Fiddler to trace HTTP activity is straightforward:
+
+#. Open Fiddler.
+#. If needed, begin capturing traffic (:menuselection:`File --> Capture Traffic`). Note that Fiddler starts in
+   capture mode when it is opened, so this step may not be necessary.
+#. Navigate to the host page URL while Fiddler is running, then reproduce the issue if needed.
+#. Once the issue is reproduced, save the Fiddler session as an archive
+   (:menuselection:`File --> Save --> All sessions...`). The resulting file should have the file extension ``.saz``.
+
+
+Session IDs
+-----------
+
+Whenever an :ref:`action URL <Action URLs>` is navigated to, Office Online creates a unique session ID. This session
+ID allows Microsoft engineers to quickly retrieve all server logs related to that session, including information
+about the WOPI calls that were made to the host. The session ID is passed back in the WOPI action URL HTTP response in
+the **X-UserSessionId** response header.
+
+The easiest way to retrieve the session ID is to use Fiddler, as described previously. However, you can also use the
```

Comment:
It might be worth noting that SessionID is also sent by the client on every request, in request headers.